### PR TITLE
Integrate objecstore file and fix get_objects > 2gb

### PR DESCRIPTION
**Pull request by roel-devries**

### CODE REVIEW

File: gobcore/datastore/objectstore.py [hunk 1]
```diff
@@ -1,13 +1,151 @@
...
 from gobcore.exceptions import GOBException
 
+LOG = logging.getLogger('objectstore')
+
+logging.getLogger("requests").setLevel(logging.WARNING)
+logging.getLogger("urllib3").setLevel(logging.WARNING)
+logging.getLogger("swiftclient").setLevel(logging.WARNING)
+
+
+def make_config_from_env():
```
Comment (fzonneveld-adam) [inline gobcore/datastore/objectstore.py:54]: Only used local prefix _

Comment (fzonneveld-adam) [reply gobcore/datastore/objectstore.py:54]: Create a function for this does not help to make the dict immutable.

File: gobcore/datastore/objectstore.py [hunk 2]
```diff
@@ -1,13 +1,151 @@
...
+    # until all objects will be fetched
+    while len(objects) == limit:
+        kwargs['marker'] = objects[-1]['name']
+        _, objects = conn.get_container(container, **kwargs)
+        for object_info in objects:
+            yield object_info
+
+
+def get_object(connection, object_meta_data: dict, dirname: str, chunk_size: int = 50_000_000,
+               max_chunks: int = None, **kwargs) -> bytes:
```
Comment (fzonneveld-adam) [inline gobcore/datastore/objectstore.py:113]: max_chunks never used, remove, simplifies everything.

File: gobcore/datastore/objectstore.py [hunk 3]
```diff
@@ -1,13 +1,151 @@
...
+        tenant_name=store['TENANT_NAME'],
+        auth_version=store['VERSION'],
+        os_options=os_options
+    )
+
+    return connection
+
+
+def get_full_container_list(conn, container, **kwargs) -> list:
+    limit = kwargs.get('limit', 10000)
```
Comment (fzonneveld-adam) [inline gobcore/datastore/objectstore.py:96]: Put this in de argument list met een default value of 10_000

**Comment by jasper-jager** [comment]
Shouldn't this be fixed in the objectstore library? Might benefit other teams as well

**Review by jasper-jager — APPROVED** [review]

**Pull request merged**